airflow/contrib/operators/sqoop_operator.py

```diff
@@ -17,75 +17,160 @@
 This module contains a sqoop 1 operator
 """
 
+from airflow.contrib.hooks.sqoop_hook import SqoopHook
+from airflow.exceptions import AirflowException
 from airflow.models import BaseOperator
 from airflow.utils.decorators import apply_defaults
-from airflow.contrib.hooks.sqoop_hook import SqoopHook
 
 
 class SqoopOperator(BaseOperator):
     """
     execute sqoop job
     """
```

Comment:
can we also add template fields so we can see them in the UI

Reply:
Yes